### DISCUSSION TITLE
Dumping Famicom games on EPROMs

### DISCUSSION THREAD

**Discussion by ghost** [sanni/cartreader#269] [Need help]
I have a couple of Famicom development (??) boards I want to dump but want to make sure it is possible before I mess anything up.
One board has chips with the text "テトリス" (Tetris) written on the stickers. The chips are what appear to be a Mitsubishi M5L27256K and a Mitsubishi M5L2764K. The second board has the text "アルカノイド" (Arkanoid) and has the Mitsubishi M5L27256K plus a Fujitsu MBM2764-25. 
The NesCartDB reports these games usually use much smaller chips. Will I run into any problems dumping these with the cart reader? Will I just need to use trial and error for the correct settings when dumping?
Thanks in advance for any help!

**Comment by sanni** [comment]
I don't think anyone has tried that before. If the dev boards have the same mapper as the retail ones it could work.

**Reply by ghost** [reply]
These boards are quite strange, they each have: a toggle switch, a 6pin connector on the top, and two bodge wires leading from the connector pins, and up to a spot where they cut a trace. I believe I will need to de-socket the chips and use some sort of EPROM dumper to dump the full contents? 

Thanks for the reply and your time!

**Reply by StanMOI** [reply]
Better to show us pictures ;)

**Comment by ghost** [comment]
Here is front and back of the 3rd board which is empty
![PXL_20211008_1006436442](https://user-images.githubusercontent.com/86126028/136712509-bfa2dffd-60e6-40a7-b5bd-b4bdb24619ef.jpg)
![PXL_20211008_1005598192](https://user-images.githubusercontent.com/86126028/136712510-c5932bf3-99fb-4f64-930e-96a5c84fdbb3.jpg)

And the two boards with chips I'd like to figure out how to dump and release
![GFQ134Xt](https://user-images.githubusercontent.com/86126028/136712512-118ba507-100a-49e0-a551-9f9e4594cbf9.jpg)

**Reply by StanMOI** [reply]
OMG 😳 what a bomb ?!
